src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -150,5 +151,10 @@ public GeneratedTagHelperContext()
         /// Contains information about in-scope tag helpers, HTML attributes, and the tag helpers' output.
         /// </remarks>
         public string ExecutionContextTypeName { get; set; }
+
+        /// <summary>
+        /// The name of the property used to encode HTML. Expected to be of type <c>IHtmlEncoder</c>.
```

Comment:
Remove `Expected to be of type <c>IHtmlEncoder</c>.`, depending on how they configure the `GeneratedTagHelperContext` it may not be a requirement.